kotskinds/apis/kots/v1beta1/license_types.go

```diff
@@ -99,7 +99,7 @@ type EntitlementField struct {
 
 // LicenseSpec defines the desired state of LicenseSpec
 type LicenseSpec struct {
-	Signature         []byte                      `json:"signature"`
+	Signature         string                      `json:"signature"`
```

Comment:
if you keep these as []byte, you won't have to do `base64.StdEncoding.DecodeString(license.Spec.Signature)`

Reply:
noted, will update

Reply:
You'd still need a base64 decode of some form though?

Reply:
apparently not